src/java/org/apache/cassandra/tools/NodeTool.java

```diff
@@ -163,6 +163,7 @@ public int execute(String... args)
                 InvalidatePermissionsCache.class,
                 InvalidateRolesCache.class,
                 InvalidateRowCache.class,
+                IsDecommissionFailed.class,
```

Comment:
@jaydeepkumar1984 where is this class? I do not see it implemented as part of this PR?

Reply:
right, I think this patch may fail to compile. It is need for us to executed  test locally before post the pr.